R/autoplot.summary.seroincidence.by.R

```diff
@@ -10,12 +10,16 @@
 #' @return a [ggplot2::ggplot()] object
 #' @export
 #' @examples
+#'
 #' library(dplyr)
 #' library(ggplot2)
 #'
+#'
 #' xs_data = load_pop_data("https://osf.io/download//n6cp3/") %>%
 #'   clean_pop_data()
-#' curve = load_curve_params("https://osf.io/download/rtw5k/" )
+#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:100, .by=antigen_iso) #reduce dataset for the purposes of this example
```

Comment:
- [ ] again see https://style.tidyverse.org/syntax.html#infix-operators, and also [the section on comments](https://style.tidyverse.org/syntax.html#infix-operators:~:text=Each%20line%20of%20a%20comment%20should%20begin%20with%20the%20comment%20symbol%20and%20a%20single%20space%3A%20%23) (I won't flag remaining instances, but do this everywhere)